include/Covariances/ACovFunc.hpp

```diff
@@ -54,7 +54,7 @@ class GSTLEARN_EXPORT ACovFunc : public AStringable
   virtual bool         hasSpectrumOnSphere() const { return false; }
   virtual bool         hasSpectrumOnRn()     const { return false; }
   virtual bool         hasMarkovCoeffs()     const { return false; }
-  virtual double       normalizeOnSphere(int n = 50, double scale = 1.) const { return 1.;}
+  virtual double       normalizeOnSphere(int /*n*/ = 50, double /*scale*/ = 1.) const { return 1.;}
```

Comment:
roxygen2 seems to dislike these default arguments with commented variable names.